services/business/checkers/pods/label_presence_checker.go

```diff
@@ -0,0 +1,34 @@
+package pods
+
+import (
+	"github.com/kiali/kiali/services/models"
+	"k8s.io/api/core/v1"
+)
+
+type LabelPresenceChecker struct {
+	Pod *v1.Pod
+}
+
+// A Checker checks individual objects and builds an IstioCheck whenever the check fails.
+// SidecarPresenceChecker checks if the current Pod has an Istio Sidecar installed.
+func (checker LabelPresenceChecker) Check() ([]*models.IstioCheck, bool) {
+	pod := models.Pod{}
+	pod.Parse(checker.Pod)
+	validations := make([]*models.IstioCheck, 0)
+	valid := true
+	if _, appLabel := pod.Labels["app"]; !appLabel {
+		//do something here
+		check := models.BuildCheck("Pod has no app label", "warning", "")
+		valid = false
+		validations = append(validations, &check)
+	}
+
+	if _, versionLabel := pod.Labels["version"]; !versionLabel {
```

Comment:
String "version" should not be hard-coded but taken from config (`VersionLabelName`)